terraform/testcases/prometheus_static_awsprw/otconfig.tpl

```diff
@@ -0,0 +1,24 @@
+receivers:
+  prometheus:
+    config:
+      global:
+        scrape_interval: 15s
+      scrape_configs:
+      - job_name: "test-prometheus-sample-app"
+        static_configs:
+        - targets: [ ${sample_app_listen_address_host}:${sample_app_listen_address_port} ]
+exporters:
+  awsprometheusremotewrite:
+    endpoint: ${cortex_instance_endpoint}/api/v1/remote_write
+    aws_auth:
+      region: ${region}
+      service: "aps"
```

Comment:
Same question as before, should this be using `awsprw`?

Reply:
See [here](https://github.com/aws-observability/aws-otel-test-framework/pull/606#discussion_r840788037)